cicd/gitops/working-with-openshift-gitops.adoc

```diff
@@ -0,0 +1,18 @@
+[id="working-with-openshift-gitops"]
+= Working with OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: working-with-openshift-gitops
+toc::[]
+
+//Concept Module
+include::modules/overview-of-openshift-gitops.adoc[leveloffset=+1]
+
+//Procedure Module
+include::modules/getting-started-openshift-gitops.adoc[leveloffset=+1]
```

Comment:
Add // at the beginning of this include statement since it is not added in the same PR, or remove it, you can add it in the other PR.